src/components/post-list.tsx

```diff
@@ -32,6 +32,7 @@ export const PostList: FC<PostListProps> = async ({
           avatarSrc={post.creator?.avatarUrl ?? undefined}
           comments={post.replies ?? 0}
           likes={post.likes ?? 0}
+          liked={post.likedBySelf === true}
```

Comment:
Could we do this? 🤔
I don't know the type exactly

```suggestion
          liked={post.likedBySelf}
```